CacheManager.py

```diff
@@ -0,0 +1,290 @@
+import sqlite3
+import tempfile
+import os
+import urllib.request
+
+'''
+    This class manages local cache
+'''
+
+class CacheManager():
+    def __init__(self):
+        self.tempDir = tempfile.gettempdir()
+        self.dbName = "amigocloud_local_db.db"
+        self.tempPath = os.path.join(self.tempDir, self.dbName)
+        self.isDev = False
+
+    def devPrint(self,content):
```

Comment:
Names of the functions should not have capital letter, it should look like this: dev_print()

Please see Python naming conventions:
https://www.python.org/dev/peps/pep-0008/#naming-conventions